### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by Sharmaryan**
1. added toast
2. added modal
3. added text utilities

Screenshot (75)

Screenshot (74)

Screenshot (73)
Screenshot (76)
Screenshot (77)
Screenshot (78)
Screenshot (79)
Screenshot (80)

### CODE REVIEW

**Review by sonishreyas — APPROVED** [review]
The code looks good and clean. There are some whitespaces and commented code you can remove. And you can also use preetier here. rest it looks good

File: styles/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  bottom: 1rem;
+  left: 2rem;
+}
+.toast-text {
+  display: inline;
+}
+.toast-close {
+  display: inline;
+  margin-left: auto;
+  margin-left: 4rem;
```
Comment (harshalini) [inline styles/toast/toast.css:19]: Remove margin-left: auto as it is getting overridden

File: styles/text/text.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.text-gray {
+    color: var(--gray);
+}
+
+.text-line {
+    text-decoration: line-through;
+}
+
```
Comment (harshitpaliwal95) [inline styles/text/text.css:46]: good naming convention

**Review by harshitpaliwal95 — APPROVED** [review]
good file folder structure 
everthing looks good you can merge

**Review by harshalini — APPROVED** [review]
Good code, it is nicely understandable

**Review by agrawalshanu708 — APPROVED** [review]
use prettier to format code
good to follow css naming methods
UI SUGGESTION:
set the width of sidebar less

**Pull request merged**